packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/sync/SyncSession.kt

```diff
@@ -73,6 +74,16 @@ public interface SyncSession {
      */
     public val state: State
 
+    /**
+     * The [SyncConfiguration] responsible for controlling the session.
```

Comment:
We could use a comment explaining in what conditions will it throw.

Reply:
Suggestion:
```kt
/**
 * The [SyncConfiguration] responsible for controlling the session.
 * 
 * @throws IllegalStateException if accessed from inside a [SyncSession.ErrorHandler] due to session errors.
 */
```